charts/internal/seed-controlplane/charts/csi-vsphere/templates/vpa-vsphere-csi-controller.yaml

```diff
@@ -1,15 +1,61 @@
 ---
-apiVersion: autoscaling.k8s.io/v1beta2
+apiVersion: autoscaling.k8s.io/v1
 kind: VerticalPodAutoscaler
 metadata:
   name: vsphere-csi-controller-vpa
   namespace: {{ .Release.Namespace }}
 spec:
   resourcePolicy:
     containerPolicies:
-      - containerName: '*'
-        minAllowed:
-          memory: 20Mi
+    - containerName: vsphere-csi-controller
+      minAllowed:
+        memory: 20Mi
+      maxAllowed:
+        cpu: {{ .Values.resources.controller.maxAllowed.cpu }}
+        memory: {{ .Values.resources.controller.maxAllowed.memory }}
+      controlledValues: RequestsOnly
```

Comment:
I don't think we should be switching on `controlledValues: RequestsOnly` here. We didn't adjust the Limits to the values we've actually seen in production, so telling VPA to not scale Limits is pretty dangerous here.